docs/api.yaml

```diff
@@ -0,0 +1,1061 @@
+openapi: "3.0.3"
```

Comment:
Add a `README.md` to the repository in which you indicate that the API can be browsed at https://petstore.swagger.io/?url=https://raw.githubusercontent.com/hei-school/hei-admin-api/docs/ha-api-spec/docs/api.yaml

That link renders the yaml definition into a read-only html (user-friendly) file